services/src/modules/plugins/transform-resource-group.spec.ts

```diff
@@ -20,6 +20,7 @@ const resourceGroup: ResourceGroup = {
   upstreams: [],
   upstreamClientCredentials: [],
   policies: [],
+  remoteSchemas: [],
```

Comment:
Should we initialize this as empty here? It is a "calculated" field like policy attachment so we should be able to handle cases where it's missing (e.g. if there are no gql directives at all in the schema)